README.md

```diff
@@ -21,11 +21,11 @@ These are the repositories for the project:
 ## Install
 ### On Host System
 ##### Requirements
-At the moment, only source installation is supported. Use Ubuntu Focal.
+At the moment, only source installation is supported. Use Ubuntu Jammy.
 
-1. Install [ROS 2 Galactic](https://docs.ros.org/en/galactic/index.html)
+1. Install [ROS 2 Humble](https://docs.ros.org/en/humble/index.html)
```

Comment:
We might want to note here that the repo is tested using Cyclone DDS. They don't have to use it, but we should clarify that's our setup. I've just been told today that other projects like MoveIt2 and nav2 do this too.
```
# for cyclonedds rmw implementation
sudo apt install -y ros-humble-rmw-cyclonedds-cpp
export RMW_IMPLEMENTATION=rmw_cyclonedds_cpp
```